src/tests/core.py

```diff
@@ -259,6 +260,30 @@ def test_insert_method_msgid(self):
             '''select typeof(msgid) from sent where ackdata=?''', result)
         self.assertEqual(column_type[0][0] if column_type else '', 'text')
 
+    def test_old_knownnodes_pickle(self):
+        """Testing old(v.0.6.2) version knownnodes.dat file"""
+        is_pass = False
+        try:
+            self.load_old_knownnodes_file()
+            knownnodes.readKnownNodes()
+            is_pass = True
+        except (AttributeError, Exception) as e:
+            print('Error: ', e)
+            self.cleanup()
```

Comment:
`cleanup('knownnodes.dat')` The cleanup function is imported from module `common`

Reply:
Also maybe it's better to use `self.fail('Failed to load knownnodes: %s' % e)` here instead of `self.assertTrue(is_pass)`? What other `Exception` do you expect, I don't understand.